packages/varlet-ui/src/space/__tests__/index.spec.js

```diff
@@ -2,186 +2,97 @@ import VarSpace from '../Space'
-      [VarSpace.name]: VarSpace,
-    },
-    template,
+      expect(wrapper.find('.var-space').attributes('style')).toContain('justify-content: ' + justify)
+      wrapper.unmount()
+    })
   })
 
-  await delay(0)
-
-  await wrapper.setProps({ direction: 'row' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ direction: 'column' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ size: 'mini' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ size: 'small' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ size: 'normal' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ size: 'large' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ size: '10px' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ size: '10rem' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ size: '[10px,20px]' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ wrap: true })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ wrap: false })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ justify: 'start' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ justify: 'end' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ justify: 'center' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ justify: 'space-between' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ justify: 'space-around' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ align: 'start' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ align: 'center' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ align: 'end' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ align: 'stretch' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ align: 'baseline' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ align: 'initial' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ align: 'inherit' })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ inline: true })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  await wrapper.setProps({ inline: false })
-  expect(wrapper.html()).toMatchSnapshot()
-
-  wrapper.unmount()
-})
+  test('test space size', () => {
+    let margin = -2
+    ;['mini', 'small', 'normal', 'large'].forEach((size, index) => {
+      const wrapper = mount(VarSpace, {
+        props: { size },
+      })
+
+      expect(wrapper.find('.var-space').attributes('style')).toContain(
+        `margin: ${index === 3 ? (margin -= 1) : margin}px 0px;`
+      )
+      margin -= 1
+      wrapper.unmount()
+    })
+  })
```

Comment:
I think it would be better to use explicit dimension values for this part

Reply:
ok, i'll deal with it later.